src/transform.ts

```diff
@@ -157,9 +163,21 @@ export function transformClassPair([className, { properties, required }]: Pair):
   };
 }
 
+const validEnumKeyRegex = /[a-z]+[a-z\d_]*/i;
```

Comment:
Anchors are missing, so it really only validates one letter.
For efficiency, I think better to omit the initial `+`.